reports/pharmacy_first_report.Rmd

```diff
@@ -767,5 +768,116 @@ plot_measures(
   y_label = "Number of codes for PF conditions"
 ) + scale_color_manual(values = ethnicity_palette)
 ```
+```{r, message=FALSE, warning=FALSE, echo = FALSE}
+df_bsa_validation <- read_csv(here("lib", "validation", "data", "pf_consultation_validation_data.csv"))
+# Multiply by 0.4 to get 40% of data
+df_bsa_validation$count = df_bsa_validation$count * 0.4
 
+# Our data for clinical conditions into a tidy df
+df_opensafely <- df_measures %>%
+  filter(measure_desc == "clinical_condition") %>%
+  filter(interval_start >= as.Date("2024-02-01") & interval_start <= as.Date("2024-07-30")) %>%
+  filter(is.na(group_by)) %>%
+  select(date = interval_start, consultation_type = measure, count = numerator)
+
+# Add a new column to each data frame to identify the source
+df_opensafely <- df_opensafely %>%
+  mutate(source = 'OS')
+df_bsa_validation <- df_bsa_validation %>%
+  mutate(source = 'BSA')
+
+# Combining rows from OS and BSA dataframes
+df_validation_condition_counts <- bind_rows(df_opensafely, df_bsa_validation)
+
+# Round the 'count' column to 0 decimal places (nearest whole number)
+df_validation_condition_counts$count <- as.numeric(df_validation_condition_counts$count)
+df_validation_condition_counts$count <- round(df_validation_condition_counts$count, digits = 0)
+
+# format as MM-YYYY
+df_validation_condition_counts <- df_validation_condition_counts %>%
+  mutate(month = format(as.Date(date), "%m-%Y"))
```

Comment:
I think it'd be clearer to have these transformations all at the top together with calculating 40%?